libs/features/mas/web-components/src/utils.js

```diff
@@ -138,22 +138,27 @@ export function getTextNodes(element) {
     return textNodes;
 }
 
+
 /**
  * Helper function to create an element with attributes
  * @param {string} tag
  * @param {Object} attributes
- * @param {*} innerHTML
+ * @param {*} content
  * @returns {HTMLElement}
  */
-export function createTag(tag, attributes = {}, innerHTML) {
-    const element = document.createElement(tag);
-    element.innerHTML = innerHTML;
+export function createTag(tag, attributes = {}, content) {
```

Comment:
is it possible to use Milo's createTag method?

Reply:
since the `web-components` will be delivered to non Milo consumers as well, I didn't want to put Milo dependencies in the bundle.
In theory, esbuild should be able to tree-shake and only bundle `createTag` from Milo but I haven't time to try it out.